File: 1 - Iniciante/C99/BEE1036.c
#include <stdio.h>
#include <math.h>

int main(){
	double a, b, c;
	double delta;
	
	//Entrada de Dados
	scanf("%lf %lf %lf", &a, &b, &c);
	
	//Definição do delta
	delta = pow(b, 2) - 4*a*c;
	
	//Aplicação de Bhaskara	
	if(a==0 || delta<0){
		printf("Impossivel calcular\n");
	} 
	else{
		printf("R1 = %.5lf\n", (-b + sqrt(delta))/ (2 * a)); 
		printf("R2 = %.5lf\n", (-b - sqrt(delta))/(2 * a)); 		
	}	
}

File: 1 - Iniciante/Python 3.11/BEE1036.py
from math import sqrt

stream = input().split(" ")
a, b, c = stream
a, b, c = float(a), float(b), float(c)

delta = pow(b, 2) - (4 * a * c)
if (delta<0) or (a==0):
    print('Impossivel calcular')
else:
    x1 = (-b+sqrt(delta)) / (2*a) 
    x2 = (-b-sqrt(delta)) / (2*a)

    print(f'R1 = {x1:.5f}')
    print(f'R2 = {x2:.5f}')